hooks/pre/35-change-evr-extension-ownership.rb

```diff
@@ -0,0 +1,26 @@
+# In Katello 4.15, the 'evr' extension is removed from PostgreSQL and integrated into the Katello database via a migration.
+# This hook ensures the 'evr' extension's ownership is transferred to the 'foreman' user so migrations can act on it.
+
+if local_postgresql? && execute("rpm -q postgresql-evr", false, false)
+  is_postgresql_active = execute_command("systemctl is-active postgresql", false, true)&.first&.strip == "active"
```

Comment:
Isn't it better to check the exit code is 0 instead of parsing the output? I have no idea if systemctl has localization, but that's one example where it could break.

Reply:
Ack..Updated to check exit status code